src/late_multicellular_stage/editor/MetaballBodyEditorComponent.cs

```diff
@@ -393,6 +422,22 @@ public bool ShowMetaballOptions()
         return true;
     }
 
+    public override void SetEditorWorldTabSpecificObjectVisibility(bool shown)
+    {
+        SetEditorWorldGuideObjectVisibility(shown);
```

Comment:
Shouldn't this also call the base method or is there a reason why this doesn't?

Reply:
Well, the base method does almost the same operations as this one, except for applying preview mode. So I don't think it really is needed to call it.

Reply:
Done